Ingest/AppSettings.cs

```diff
@@ -54,6 +60,17 @@ public string Archive
         public bool RunInForeground { get; set; }
         [CategoryAttribute("Run")]
         public bool RunAsService { get; set; }
+        [CategoryAttribute("Run")]
+        public string TaskName
+        {
+            get
+            {
+                if(conf.SettingsPath == conf.DefaultSettingsPath) {
+                    return "BBCIngest"; 
+                }
+                return "BBCIngest-"+shortUid(settingsPath);
```

Comment:
The function "shortUid" also seems to prepend "BBCIngest-" to it's return value, so this will result in a value "BBCIngest-BBCIngest-0123456789ABCDEF".
Can we remove the duplication in the name?